### DISCUSSION TITLE
Issue with Pip install

### DISCUSSION THREAD

**Discussion by krinetic1234** [sebp/scikit-survival#181] [General]
@sebp I have a mac and I am getting an error here when I pip install, what is the reason for this? I'm able to easily install on google colab. Also, I installed with conda and it worked but when I run code it just says "sksurv" not found.

Here is error I get with pip install scikit-survival

ERROR: Failed building wheel for scs
  Running setup.py clean for scs
Failed to build scikit-survival ecos scs
ERROR: Could not build wheels for scikit-survival which use PEP 517 and cannot be installed directly

**Comment by sebp** [comment]
It probably can't build a third party dependency, likely osqp, which 
requires cmake. Can you please run pip install with "-v" to get more 
output from the install process?

**Reply by krinetic1234** [reply]
@sebp i installed cmake, still didn't work. pip -v gives me so much stuff, but here's where the core error is if that helps.

<img width="1013" alt="Screen Shot 2021-02-28 at 11 16 32 PM" src="https://user-images.githubusercontent.com/56781484/109464123-00bca200-7a1b-11eb-81c4-eb02448f813f.png">

aside from this, though, i installed anaconda and used the `conda install -c sebp scikit-survival` command. that command worked but when i ran my flask app, it threw a MODULE NOT FOUND error, saying that it couldn't fine 'sksurv'.

**Reply by krinetic1234** [reply]
<img width="427" alt="Screen Shot 2021-02-28 at 11 23 54 PM" src="https://user-images.githubusercontent.com/56781484/109464771-0e265c00-7a1c-11eb-99fa-cf51d8362383.png">

that's the module not found error

**Comment by sebp** [comment]
The error indicates that it failed to install ecos and scs, which are 
dependencies of scikit-survival. Can you check the output for any errors 
related to those two packages?

**Reply by krinetic1234** [reply]
Ok, thanks. Here are all the errors I'm getting.
<img width="1023" alt="Screen Shot 2021-02-28 at 11 30 57 PM" src="https://user-images.githubusercontent.com/56781484/109465421-03b89200-7a1d-11eb-9b94-49ecb680e4b3.png">
<img width="1045" alt="Screen Shot 2021-02-28 at 11 31 10 PM" src="https://user-images.githubusercontent.com/56781484/109465445-0b783680-7a1d-11eb-918e-1dfe6a8dc7b2.png">
<img width="1087" alt="Screen Shot 2021-02-28 at 11 31 38 PM" src="https://user-images.githubusercontent.com/56781484/109465481-1b901600-7a1d-11eb-8e4c-5df34bcd5ffb.png">
<img width="1069" alt="Screen Shot 2021-02-28 at 11 31 50 PM" src="https://user-images.githubusercontent.com/56781484/109465495-22b72400-7a1d-11eb-8926-3f6df6ba24fb.png">

**Reply by krinetic1234** [reply]
@sebp these are all the lines i saw highlighted in red, and i think they are all related to ecos and scs

**Comment by sebp** [comment]
The "xcrun error" indicates that something's wrong with the Xcode 
command line tools. You might have re-install it, see 
https://scikit-survival.readthedocs.io/en/latest/install.html#from-source

**Reply by krinetic1234** [reply]
Great idea @sebp, thank you. I do notice that I often receive errors from Xcode that I haven't installed all dependencies. Will do so ASAP, and let you know if I still have errors. Thanks again, and this is a great library :)

**Reply by krinetic1234** [reply]
@sebp, install works now! thank you so much
